src/promptflow/tests/test_configs/functions/class_init_complicated_ports/hello.py

```diff
@@ -0,0 +1,28 @@
+from dataclasses import dataclass, field
+
+from promptflow._sdk.entities import AzureOpenAIConnection
+
+
+@dataclass
+class ComplicatedOutput:
```

Comment:
May also give TypeDict a test in next PR

Reply:
TypeDict is not supported now to my understanding? so there will be a feature PR instead of a test only